### PR TITLE
✨ Source Apple Ads Search : add timezone from spec to request parameters 

### PR DESCRIPTION
**Pull request by Abuton**
I'm adding timezone to spec to allow for proper data aggregation as stated [here](https://developer.apple.com/documentation/apple_search_ads/reportingrequest)
image

## How
I have added `timezone` as one of the optional configuration a user needs to fill when setting up a connection with apple ads search source connector. I have also preserved the default that was included when this source connector was initially developed `UTC`.

## Recommended reading order
1. `airbyte/airbyte_integrations/connectors/source-apple-search-ads/source_apple_search_ads/spec.yaml`
2. `airbyte/airbyte_integrations/connectors/source-apple-search-ads/source_apple_search_ads/manifest.yaml`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

There are no breaking changes in this Pull Request.

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
@Abuton do you have a link to Apple Ads documentation showing what timezones are supported?

**Comment by Abuton** [comment]
> @Abuton do you have a link to Apple Ads documentation showing what timezones are supported?

Yes, here it is: https://developer.apple.com/documentation/apple_search_ads/reportingrequest

File: airbyte-integrations/connectors/source-apple-search-ads/source_apple_search_ads/manifest.yaml [hunk 1]
```diff
@@ -113,7 +113,7 @@ definitions:
         request_body_json:
           startTime: "{{ stream_slice.start_time }}"
           endTime: "{{ stream_slice.end_time }}"
-          timeZone: "UTC"
+          timeZone: "{{ config.time_zone }}"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-apple-search-ads/source_apple_search_ads/manifest.yaml:116]:
```suggestion
          timeZone: "{{ config.get('time_zone', "UTC") }}"
```
Using this method it won't introduce a breaking change to existing connector.

File: docs/integrations/sources/apple-search-ads.md [hunk 2]
```diff
@@ -20,7 +20,8 @@ This page contains the setup guide and reference information for the Apple Searc
 5. For **Org Id**, enter the Id of your organization (found in the Apple Search Ads UI).
 6. Enter the **Client ID** and the **Client Secret** from [Step 1](#step-1-set-up-apple-search-ads).
 7. For **Start Date** and **End Date**, enter the date in YYYY-MM-DD format. For DAILY reports, the Start Date can't be earlier than 90 days from today. If the End Date field is left blank, Airbyte will replicate data to today.
-8. Click **Set up source**.
+8. You can also set the timezone to determine how the report data is aggregated
```
Comment (marcosmarxm) [inline docs/integrations/sources/apple-search-ads.md:23]: Add a note saying default is UTC.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: docs/integrations/sources/apple-search-ads.md (hunk 2 above)
Comment (Abuton) [reply docs/integrations/sources/apple-search-ads.md]: Done

**Pull request closed without merging**